### ISSUE TITLE
Every time when there is Post with that there is a Get request getting called automatically in server

### ISSUE THREAD

**Issue by pritamkumarpanda1** [dotnet/wcf#4989]
Hello Team,
I am facing one issue with WCF. I migrated my project from .Net 4.7 framework to .Net 6 with help of CoreWcf. When calling the service through client, I am getting **GET** issue every time when there is **POST** getting called in server. Locally when I test through SOAP UI, I didn't get any issue but in server it does. I want to stop the GET call as every time it is automatically getting called. I have tried with WebInvoke in Operation Contract and making changes in "serviceMetadataBehavior.HttpGetEnabled" but nothing worked. I have attached the app-insights of the request. Any suggestions please?

https://github.com/CoreWCF/CoreWCF/issues/977

![get](https://user-images.githubusercontent.com/23519276/212613884-0fe3c3e5-ff54-4173-86c6-e7a3977ce805.PNG)

**Comment by mconnew** [comment]
Is your service a REST-ish service using WebHttpBinding? Is the client .NET Framework or .NET [Core]? I double checked the code in WCF as we used to do an extra request a while ago to work around lack of Expect 100-Continue support, but we pulled that code out a long time ago and that was sending an extra HEAD request, not GET. I'm not aware of where that GET request is coming from. If the client is .NET, if you turn on intellitrace tracing in VS it logs every HTTP request which is sent. Are you seeing the GET requests coming from the client side?
